libs/blocks/caas/utils.js

```diff
@@ -358,7 +358,9 @@ export const getConfig = async (originalState, strs = {}) => {
         state.endpoint
       }${targetActivity}?originSelection=${originSelection}&contentTypeTags=${state.contentTypeTags.join(
         ',',
-      )}&collectionTags=${collectionTags}&excludeContentWithTags=${excludeContentWithTags}&language=${language}&country=${country}&complexQuery=${complexQuery}&excludeIds=${excludedCards}&currentEntityId=&featuredCards=${featuredCards}&environment=&draft=${
+      )}&secondSource=${state.showSecondarySource ? state.secondSource.join(',') : []}&secondaryTags=${state.showSecondarySource ? state.secondaryTags.join(
+        ',',
```

Comment:
The spacing seems a little off

Reply:
every time I try to mess around with the way this is spaced I get linting errors - so maybe I haven't tried the correct way to space it yet?

Reply:
I dealt with that too - essentially there aren't any places for line breaks except the join statements.  I think it's weird but fine.